packages/block-library/src/site-logo/style.scss

```
@@ -20,6 +20,12 @@
 		max-width: 100%;
 	}
 
+	&.has-custom-border {
+		img {
+			box-sizing: border-box;
+		}
+	}
```

```suggestion
	&.has-custom-border img {
		box-sizing: border-box;
	}
```

optional, but it's a little less work for the SASS compiler :)